Bond.podspec

```diff
@@ -21,15 +21,15 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = "8.0"
   s.osx.deployment_target = "10.10"
   s.tvos.deployment_target = "9.0"
-  s.source       = { :git => "https://github.com/SwiftBond/Bond.git", :tag => "6.7.0" }
+  s.source       = { :git => "https://github.com/SwiftBond/Bond.git", :tag => "6.8.0" }
   s.source_files  = "Sources/**/*.{h,m,swift}", "Supporting Files/Bond.h"
   s.ios.exclude_files = "Sources/Bond/AppKit"
   s.tvos.exclude_files = "Sources/Bond/AppKit"
   s.osx.exclude_files = "Sources/Bond/UIKit"
   s.pod_target_xcconfig = { "OTHER_SWIFT_FLAGS" => "-DBUILDING_WITH_XCODE $(inherited)" }
   s.requires_arc = true
 
-  s.dependency "ReactiveKit", "~> 3.8"
+  s.dependency "ReactiveKit", "~> 3.9"
   s.dependency "Differ", "~> 1.0"
```

Comment:
You'll want to move Differ up to `~> 1.1.0` for the Xcode 9.3/Swift 4.1 compatibility. The Podspec is already deployed to CocoaPods.